src/status_im/ui/screens/privacy_and_security_settings/views.cljs

```diff
@@ -89,3 +103,19 @@
           :on-press            #(re-frame/dispatch [:navigate-to :delete-profile])
           :accessibility-label :dapps-permissions-button
           :chevron             true}])]]))
+
+(defn radio-item [id value]
+  [quo/list-item
+   {:active    (= value id)
+    :accessory :radio
+    :title     (get titles id)
+    :on-press  #(re-frame/dispatch [:multiaccounts/profile-picture-show-to-switched id])}])
```

Comment:
Wrong event name here:
`multiaccounts.ui/profile-picture-show-to-switched`, as you defined it in 
https://github.com/status-im/status-react/pull/11768/files#diff-c86795f915dded69102eae21e324dbf14197b79a635d3123611a7ee5ea926770R154